addons/binding/org.openhab.binding.enocean/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="enocean"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- RepeaterMode channel -->
+	<channel-type id="repeaterMode" advanced="true">
+		<item-type>String</item-type>
+		<label>Repeater Mode</label>
+		<description>Repeater mode of EnOcean gateway/device</description>
+		<state>
+			<options>
+				<option value="OFF">Repeater off</option>
+				<option value="LEVEL1">Level-1</option>
+				<option value="LEVEL2">Level-2</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="setBaseId" advanced="true">
+		<item-type>String</item-type>
+		<label>Set base Id</label>
+		<description>Set base id of EnOcean gateway. Use this with care! Must be between 0xFF800000 and 0xFFFFFF80</description>
+	</channel-type>
+
+	<!-- Light Switch Channel -->
+	<channel-type id="lightSwitch">
```

Comment:
Isn't there any numbering/naming convention in the EnOcean spec for the single values? I would expect the channel types here to directly refer to parts of the spec (and also have their IDs chosen that way).

Reply:
fixed => set to generalSwitch